src/ui_framework/components/info_panel/_info_panel.scss

```diff
@@ -0,0 +1,8 @@
+.kuiInfoPanel {
+  padding: 20px;
+  line-height: $lineHeight;
+}
+
+.kuiInfoPanel--warning {
+  background-color: #FFF6E4;
```

Comment:
Looks like you pulled the color from my screenshot. I'm no designer, and I know nothing about color theory, I just moved a slider until I found a color that looked good enough to me. I'm fine with leaving it this way, since I think it looks good enough, but just wanted to give you a little warning...